cmd/crc/cmd/start.go

```diff
@@ -68,6 +70,7 @@ func initStartCmdFlagSet() *pflag.FlagSet {
 	flagSet := pflag.NewFlagSet("start", pflag.ExitOnError)
 	flagSet.StringP(config.Bundle.Name, "b", constants.GetDefaultBundle(), "The system bundle used for deployment of the OpenShift cluster.")
 	flagSet.StringP(config.VMDriver.Name, "d", machine.DefaultDriver.Driver, fmt.Sprintf("The driver to use for the CRC VM. Possible values: %v", machine.SupportedDriverValues()))
+	flagSet.StringP(config.PullSecret.Name, "p", "", "Pull secret for User (Get it from https://cloud.redhat.com/openshift/install/metal/user-provisioned")
```

Comment:
we use the URL twice.

can we extract this? as I am also sure this 'might' change...